app/upw/templates/accessibility-statement.njk

```diff
@@ -0,0 +1,149 @@
+{% extends "common/templates/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "common/templates/components/question/macro.njk" import renderQuestion %}
+{% from "common/templates/components/widgets/macro.njk" import widgets %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set backLink = 'task-list' %}
+
+{% block pageTitle %}
+{{ pageTitle }}
+{% endblock %}
+
+{% block header %}
+{% include "common/templates/partials/header.njk" %}
+{% endblock %}
+
+{% block content %}
+{# include errorSummary partial #}
+{{ super() }}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">
+            {{ pageTitle }}
+        </h1>
+
+        <p>
+            This accessibility statement applies to the website located at <br>
+            <a href="https://sign-in.hmpps.service.justice.gov.uk/auth/sign-in?redirect_uri=https://hmpps-assessments.service.justice.gov.uk/login/callback">https://hmpps-assessments.service.justice.gov.uk</a>
```

Comment:
The href for this should just be https://hmpps-assessments.service.justice.gov.uk